### PR TITLE
Fix findpw exception handling

### PR DESCRIPTION
**Pull request by geoseong**
## Specify project
`react-native` and`server`

## Description

This PR started from SendGrid's email sent failed(I created my own email domain to test in localhost), and `react-relay`'s Error type in `onCompleted` callback didn't match with the server's error type.

According to [relay's useMutation document](https://relay.dev/docs/api-reference/use-mutation/#return-value), `onCompleted` says: "Internally errors are not allowed, CRITICAL error will be thrown in the onError handler."

so I tried to figure out how to make the server's return can be detected on the client's "onError" callback.

### Change return field type from `Boolean` to `nonNull('Boolean')`

I've noticed if the server doesn't have data that is literally null, the client always detects the server's response in the "onCompleted" callback not "onError".

**Existing way**

Untitled
Untitled (1)
Untitled (4)

**Modified**
Untitled (2)
Untitled (3)
Untitled (5)

### The way how to handle error message from server

The error from the server was different that I expected.
it's different from the graphql playground's result.

Untitled (6)

so I try to catch error message using [`includes()`](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/String/includes) fn.

## Related Issues

none

## Tests

none

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/hackatalk/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] Run `yarn lint && yarn tsc`
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

**Comment by hyochan** [comment]
@0916dhkim Could you kindly review this and give your opinion?

**Review by hyochan — APPROVED** [review]
LGTM!

File: server/src/resolvers/User/mutation.ts [hunk 1]
```diff
@@ -294,7 +294,7 @@ export const updateProfile = mutationField('updateProfile', {
 });
 
 export const findPassword = mutationField('findPassword', {
-  type: 'Boolean',
+  type: nonNull('Boolean'),
```
Comment (0916dhkim) [inline server/src/resolvers/User/mutation.ts:297]: Good catch!

**Review by 0916dhkim — APPROVED** [review]
Sorry for the delay.
This PR looks good to me. I should be more careful about Relay mutation and error handling next time.

**Pull request merged**